### ISSUE TITLE
Bug:: wrong named import from node_modules not detected

### ISSUE THREAD

**Issue by sagiavinash** [benmosher/eslint-plugin-import#655]
The mistakes in es6 named imports are detected properly only if the modules are local user-written modules. Mistakes in named imports from `node_modules` are not detected

Example:
`import React, { PropTypes } from 'react'` is the correct import statement
but
`import React, { propTypes } from 'react'` wouldnt throw an error!

my .eslintrc config is as follows, according to the docs
```
"extends": [
  "airbnb",
  "plugin:import/errors",
  "plugin:import/warnings",
],
"settings": {
  "import/external-module-folders": [],
},
```

**Comment by benmosher** [comment]
You're correct. As long as React doesn't expose ES6 versions of their entry point in their npm package (which it looks like they don't), this plugin is not equipped to lint them and they will be ignored by default.

I tried a few times to handle this, but it's a hard problem. At some point, I may try out something like what VS Code does with TypeScript typings, but you would need to install them yourself.
